sdc/functions/numpy_like.py

```diff
@@ -967,3 +967,77 @@ def getitem_by_mask_impl(arr, idx):
             return result_data
 
     return getitem_by_mask_impl
+
+
+def skew(a):
+    pass
+
+
+def nanskew(a):
+    pass
+
+
+@sdc_overload(skew)
+def np_skew(arr):
+    if not isinstance(arr, types.Array):
+        return
+
+    def skew_impl(arr):
+        len_val = len(arr)
+        nfinite = 0
+        _sum = 0
+        square_sum = 0
+        cube_sum = 0
+
+        for idx in numba.prange(len_val):
+            if numpy.isfinite(arr[idx]):
+                nfinite += 1
+                _sum += arr[idx]
+                square_sum += arr[idx] ** 2
+                cube_sum += arr[idx] ** 3
+            else:
+                return numpy.nan
+
+        n = nfinite
+        m2 = (square_sum - _sum * _sum / n) / n
```

Comment:
I believe we could move this formula in somewhere and then use in Series/Rolling/GroupBy methods

@densmirn ?